GOVERNANCE.md

```diff
@@ -0,0 +1,69 @@
+# Cloud Native Buildpacks Governance
+This document defines the governance structure for Cloud Native Buildpacks. This project is committed to building an open, inclusive, productive and self-governing open source community that maintains high-quality tools for translating source code into OCI images. The community is governed by this document with the goal of defining how said community should work together to achieve this goal.
```

Comment:
```suggestion
This document defines the governance structure for Cloud Native Buildpacks. This project is committed to building an open, inclusive, productive and self-governing open source community that maintains high-quality tools for translating source code into OCI images. This document prescribes how said community should work together to achieve this goal.
```

Not as sure about the replacement wording but `The community is governed by this document` feels wrong

Reply:
I think i copied that from an CNCF project's governance doc, but agree it's odd. I'll think on it.